GOVERNANCE.md

```diff
@@ -28,75 +27,72 @@ Flux strives to operate independently of specific partisan interests, and for de
 
 ### Code of Conduct
 
-The Flux community adheres to the CNCF Code of Conduct <https://github.com/cncf/foundation/blob/master/code-of-conduct.md>.
+[Code of Conduct](CODE_OF_CONDUCT.md) violations by community members will be discussed and resolved on the [private Maintainer mailing list](mailto:cncf-flux-maintainers@lists.cncf.io). If the reported CoC violator is a Maintainer, the Maintainers will instead designate two Maintainers to work with CNCF staff in resolving the report.
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a _Flux_ [Oversight Committee] member.
+### Meetings
 
-If no conclusion can be reached in meditation, such issues can be escalated to the CNCF mediator, Mishi Choudhary <mishi@linux.com>, in which case CNCF may choose to intervene.
+Time zones permitting, Maintainers are expected to participate in the [public
+developer meeting](README.md#meetings).
 
-## Roles in the Flux Community
+Maintainers will also have closed meetings in order to discuss security reports
+or Code of Conduct violations.  Such meetings should be scheduled by any
+Maintainer on receipt of a security issue or CoC report.  All current Maintainers
+must be invited to such closed meetings, except for any Maintainer who is
+accused of a CoC violation.
```

Comment:
Security related issues are often initially only shared on a need-to-know basis, and this process will become even more complicated when we finalize the plans we are currently working on to formalize the Coordinated Vulnerability Disclosure process (likely based on https://github.com/kubernetes/committee-security-response/blob/main/private-distributors-list.md).

Reply:
The above paragraph was copied https://github.com/cncf/project-template/blob/main/GOVERNANCE-maintainer.md#meetings from and was meant to illustrate as to why internal/closed meetings might happen.

Is it OK, if I refer to https://fluxcd.io/security/ for more information on the specifics of the security process to resolve any ambiguity in that paragraph?

Reply:
Yes, this sounds good to me.

Reply:
:+1: